packages/commons/src/interop-token/interopTokenService.ts

```diff
@@ -278,4 +283,7 @@ export class InteropTokenGenerator {
 
     return `${serializedToken}.${jwtSignature}`;
   }
+
+  private convertAudience = (input: string | string[]): string | string[] =>
```

Comment:
minor, but a bit less generic
```suggestion
  private toJwtAudience = (input: string | string[]): string | string[] =>
```